src/js/components/Drop/StyledDrop.js

```diff
@@ -27,6 +27,154 @@ const dropKeyFrames = keyframes`
   }
 `;
 
+function isObject(object) {
+  return object != null && typeof object === 'object';
+}
+
+function deepEqual(object1, object2) {
```

Comment:
```suggestion
const deepEqual = (object1, object2) => {
```

Reply:
totally forgot to change this back lol thank you!